common/api-review/util.api.md

```diff
@@ -189,6 +189,20 @@ export type ExperimentalKey = 'authTokenSyncURL' | 'authIdTokenMaxAge';
 // @public
 export function extractQuerystring(url: string): string;
 
+// Warning: (ae-missing-release-tag) "FetchProvider" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
```

Comment:
This file is autogenerated. The tag needs to go in a jsdoc comment above the class in [packages/util/src/fetch_provider.ts](https://github.com/firebase/firebase-js-sdk/pull/8480/files#diff-1e985366f52403ba43ecb690e2ddd10d2eaaf1b0f1c7769e7edc5085c14f2c32)

Reply:
Updated.